tasks/copy.js

```diff
@@ -0,0 +1,4 @@
+const shell = require('shelljs')
+
+// Copy files using shx (for suppport cross-platform)
+shell.exec('shx cp -r ./src/_locales ./src/manifest.json ./_dist')
```

Comment:
It should also be possible to compress JSON by removing newlines and whitespace.

Reply:
I know `shell.cp ()`. However, I dare to do this because the cp command felt shell command notation is more readable. 💭 
Experience has shown that files to be copied often increase. Shell command gradually becomes longer... So it's more convenient to expand the array of file paths in JS, so I divided it into `copy.js`in advance.

But I thought it was pointed out now, this is a Chrome extension so it doesn't seem so much to increase the number of files to copy ....

```zsh
shx cp -r ./src/_locales ./src/manifest.json ./_dist'
```

I feel like placing this inline just like a clean task.

---

> compress JSON
Does this also mean compressing `manifest.json` at compress step?

Reply:
> Does this also mean compressing manifest.json at compress step?

Yup we can also compress the manifest.json. I've noticed a few other extensions, like google-hangouts, do this.